Orm/Xtensive.Orm/Orm/Configuration/TaggingBehavior.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2007-2022 Xtensive LLC.
+// This code is distributed under MIT license terms.
+// See the License.txt file in the project root for more information.
+// Created by: Georgi Sargsyan
+// Created:    2022.07.19
+
+namespace Xtensive.Orm.Configuration
+{
+  public enum TaggingBehavior
```

Comment:
Would be easier to understand your idea if this enum had description for default and alternative behavior

Reply:
At least raw ideas